### ISSUE TITLE
Illegal inheritance in IDEAS.Electric.Photovoltaics.Components

### ISSUE THREAD

**Issue by tbeu** [open-ideas/IDEAS#398]
```
Warning while loading the file f:\Libraries\IDEAS\IDEAS\Electric\Photovoltaics\Components\DCgen.mo.
The Modelica class of IDEAS.Electric.Photovoltaics.Components.DCgen (class) and its base class Modelica.Electrical.Analog.Interfaces.OnePort (model) are not compatible.
Message: Modelica.IncompatibleBaseClass

Warning while loading the file f:\Libraries\IDEAS\IDEAS\Electric\Photovoltaics\Components\PvVoltageToPower.mo.
The Modelica class of IDEAS.Electric.Photovoltaics.Components.PvVoltageToPower (class) and its base class Modelica.Blocks.Interfaces.BlockIcon (block) are not compatible.
Message: Modelica.IncompatibleBaseClass

Warning while loading the file f:\Libraries\IDEAS\IDEAS\Electric\Photovoltaics\Components\SimpleInverter.mo.
The Modelica class of IDEAS.Electric.Photovoltaics.Components.SimpleInverter (class) and its base class Modelica.Electrical.Analog.Interfaces.TwoPort (model) are not compatible.
Message: Modelica.IncompatibleBaseClass
```

**Comment by Mathadon** [comment]
Will look into this, thanks for reporting!

**Comment by rubenbaetens** [comment]
Solved in #399.
